extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -30,7 +31,27 @@ class AmpAccordion extends AMP.BaseElement {
     /** @const @private {!NodeList} */
     this.sections_ = this.getRealChildren();
 
+    /** @const @private {!Window} */
+    this.win_ = this.getWin();
+
+    /** @const @private {string} */
+    this.id_ = this.getSessionStorageKey_();
+
+    /** @const @private {!Object|undefined} */
+    this.currentState_ = null;
     this.element.setAttribute('role', 'tablist');
+
+    // sessionStorage key: special created id for this element, this.id_.
+    // sessionStorage value: string that can convert to this.currentState_ obj.
+    // TODO: add test for setting and reading sessionStorage.
+    try {
```

Comment:
How about moving the try-catch into `#getSessionState`? Should reduce some code.